changelog.d/1431.feature.rst

```diff
@@ -0,0 +1 @@
+Added support for Python 3.13. Reported by @rani-pinchuk (gh issue #1430). Fixed by @rani-pinchuk (gh pr #1431).
```

Comment:
Fixes 3.12 as well: https://github.com/dateutil/dateutil/issues/1430#issuecomment-3005929957 
```suggestion
Added support for Python 3.12. Reported by @rani-pinchuk (gh issue #1430). Fixed by @rani-pinchuk (gh pr #1431).
```

Reply:
I am not sure why. Maybe you mean `3.12` **and** `3.13`? (see the changes in `tox.ini`)